src/app/plugins/scores.plugin.ts

```diff
@@ -3,123 +3,146 @@ import { Plugin } from '../../common/plugin';
+            console.log(homeData);
+
+            visitorData = visitorData.split('%20');
+            homeData = homeData.split('%20');
+
+            visitorData[0] = visitorData[0].slice(1); //Remove the prefix that starts the visitors name
+
+            const visitorName = this._getTeamNameFromTeamData(visitorData);
+            const homeName = this._getTeamNameFromTeamData(homeData);
+
+            const visitorScore = parseInt(visitorData[visitorData.length - 1]);
+            const homeScore = parseInt(homeData[homeData.length - 1]);
+
+            //Boolean for if the searched team is the one winning
+            let winning = 0;
+            let opponentName = '';
+            let teamName = '';
+
+            //If searched team is the visitor, else
+            if (this._containsAllTokens(visitorName, desiredTeamQuery.split(' '))) {
+
+              if (visitorScore > homeScore) { winning = 1; }  //Winning
+              else if (visitorScore < homeScore) { winning = 0; } //Losing
+              else if (visitorScore == homeScore) { winning = 2; } //Tied
+
+              opponentName = homeName;
+              teamName = visitorName;
+            } else {
+
+              if (homeScore > visitorScore) { winning = 1; }  //Winning
+              else if (homeScore < visitorScore) { winning = 0; } //Losing
+              else if (homeScore == visitorScore) { winning = 2; } //Tied
+
+              opponentName = visitorName;
+              teamName = homeName;
+            }
+
+            let winningString;
+            switch (winning) {
+              case 0: winningString = 'losing';
+                break;
+              case 1: winningString = 'winning';
+                break;
+              case 2: winningString = 'tied';
+                break;
+              default:
+                winningString = 'unknown';
+                break;
+            }
+
+            const embed = new RichEmbed();
+            embed.setTitle(visitorName + ' at ' + homeName);
+            embed.setColor('#7289da');
+            embed.setDescription(
+              teamName + ' is currently ' + winningString + ' against ' + opponentName
+            );
+            embed.setFooter(visitorScore + ' - ' + homeScore);
+            message.channel.send(embed);
+            messagesSent++;
           }
-
-          const winningString = winning ? "winning" : "losing";
-
-          const embed = new RichEmbed();
-          embed.setTitle(visitorName + " at " + homeName);
-          embed.setColor('#7289da');
-          embed.setDescription(teamName + " is currently " + winningString + " against " + opponentName);
-          embed.setFooter(visitorScore + " - " + homeScore);
-
-          message.channel.send(embed);
-
         }
       }
 
-      if (!teamFound)
-        message.reply("Team not found!");
+      if (!teamFound) { message.reply('Team not found!'); }
 
     } catch (error) {
       console.log('Error Occured: ' + error);
     }
```

Comment:
use `this.container.LoggerService.error(...)`.

Also, buffer all the embed's and send them at the end. this way either all/none will go through; rn an error would interrupt the flow.